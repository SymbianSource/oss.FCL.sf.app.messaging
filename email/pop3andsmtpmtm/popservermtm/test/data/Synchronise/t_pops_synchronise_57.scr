+OK demon POP3 server ready (demon.co.uk)
+OK username accepted
+OK password accepted
+OK 2 2888
+OK 2 messages
1 1444
2 1444
.
-ERR UIDL not supported
-ERR Unknown Command "CAPA"
+OK TOP 2 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message013; Wed, 25 Jul 2001 13:53:48 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id aa1102725; 25 Jul 2001 13:41 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac10010254f6fc262b@teleca161.teleca.com> for <freddy@yashar.demon.co.uk>;
 Wed, 25 Jul 2001 14:56:12 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
	id <PRD792MK>; Wed, 25 Jul 2001 14:44:08 +0100
Message-ID: <t_pops_synchronise_message013@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 13
Date: Wed, 25 Jul 2001 14:37:33 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
	charset="iso-8859-1"

Thirteen Thirteen Thirteen Thirteen Thirteen Thirteen Thirteen Thirteen
.
+OK TOP 1 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message014; Wed, 25 Jul 2001 13:44:27 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id aa1103773; 25 Jul 2001 13:41 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac10010254f6fc76ad@teleca161.teleca.com> for <freddy@yashar.demon.co.uk>;
 Wed, 25 Jul 2001 14:56:33 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
	id <PRD792ML>; Wed, 25 Jul 2001 14:44:14 +0100
Message-ID: <t_pops_synchronise_message014@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 14
Date: Wed, 25 Jul 2001 14:37:38 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
	charset="iso-8859-1"

Fourteen Fourteen Fourteen Fourteen Fourteen Fourteen Fourteen Fourteen
.
** Read cancelled
+OK RETR 2 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message014; Wed, 25 Jul 2001 13:44:27 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id aa1103773; 25 Jul 2001 13:41 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac10010254f6fc76ad@teleca161.
teleca.com> for <freddy@yashar.demon.co.uk>;
 Wed, 25 Jul 2001 14:56:33 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <PRD792ML>; Wed, 25 Jul 2001 14:44:14 +0100
Message-ID: <t_pops_synchronise_message014@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 14
Date: Wed, 25 Jul 2001 14:37:38 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Fourteen Fourteen Fourteen Fourteen Fourteen Fourteen Fourteen Fourteen
Fourteen Fourteen 
Fourteen Fourteen Fourteen Fourteen Fourteen Fourteen Fourteen Fourteen
Fourteen Fourteen 
Fourteen Fourteen Fourteen Fourteen Fourteen Fourteen Fourteen Fourteen
Fourteen Fourteen 
Fourteen Fourteen Fourteen Fourteen Fourteen Fourteen Fourteen Fourteen
Fourteen Fourteen 
.
+OK RETR 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id 996069228:10:01285:41; Wed, 25 Jul 2001 13:53:48 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id aa1102725; 25 Jul 2001 13:41 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac10010254f6fc262b@teleca161.
teleca.com> for <freddy@yashar.demon.co.uk>;
 Wed, 25 Jul 2001 14:56:12 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <PRD792MK>; Wed, 25 Jul 2001 14:44:08 +0100
Message-ID: <t_pops_synchronise_message013@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 13
Date: Wed, 25 Jul 2001 14:37:33 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Thirteen Thirteen Thirteen Thirteen Thirteen Thirteen Thirteen Thirteen
Thirteen Thirteen 
Thirteen Thirteen Thirteen Thirteen Thirteen Thirteen Thirteen Thirteen
Thirteen Thirteen 
Thirteen Thirteen Thirteen Thirteen Thirteen Thirteen Thirteen Thirteen
Thirteen Thirteen 
Thirteen Thirteen Thirteen Thirteen Thirteen Thirteen Thirteen Thirteen
Thirteen Thirteen 
.
** Read cancelled
+OK disconnecting

