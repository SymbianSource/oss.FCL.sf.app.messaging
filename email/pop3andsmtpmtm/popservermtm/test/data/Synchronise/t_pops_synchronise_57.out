STAT
LIST
UIDL
CAPA
TOP 2 1
TOP 1 1
RETR 2
RETR 1
QUIT
